packages/components/src/DateTimePickers/pickers/TimePicker/TimePicker.component.test.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import { TimePicker, isBefore, addInterval, getOptions } from './TimePicker.component';
+
+describe('TimePicker component', () => {
+	it('should render', () => {
+		const wrapper = shallow(<TimePicker onSubmit={jest.fn()} />);
+
+		expect(wrapper.getElement()).toMatchSnapshot();
+	});
+	describe('functions', () => {
+		describe('function - isBefore', () => {
```

Comment:
You should not expose this internal function. This will be tested with the component.
The reason: if we refacto an internal implementation, changing its internal api, your tests will fail even if it doesn't change your component

Reply:
so I just not export these functions and remove the test cases?